diag_manager/fms_diag_yaml.F90

```diff
@@ -1979,7 +2001,8 @@ subroutine fms_diag_yaml_out(ntimes)
   enddo
   tier2size = i
 
-  call write_yaml_from_struct_3( 'diag_manifest.yaml.'//string(mpp_pe())//c_null_char,  1, keys, vals, &
+  call get_instance_filename('diag_manifest.yaml.'//string(mpp_pe()), filename)
```

Comment:
Is `get_instance_filename` adding a set appendix from FMS2_io in order to support ensembles? Would it be overwritten otherwise?

Just curious if this is something we should be doing for all output ASCII files.